config/txloader/load/harvestcraft/lang/pl_PL.lang

```diff
@@ -0,0 +1,1136 @@
+        
+item.broccolimacItem.name=Brokuł z Serem
+item.broccolindipItem.name=Brokuł z Dipem
+item.creamedbroccolisoupItem.name=Brokułowa Zupa Krem
+        
+item.sweetpotatopieItem.name=Ciasto Batatowe
+item.candiedsweetpotatoesItem.name=Kandyzowane Bataty
+item.mashedsweetpotatoesItem.name=Tłuczone Bataty
+        
+item.steamedpeasItem.name=Fasola na Parze
+item.splitpeasoupItem.name=Grochówka
+        
+item.pineappleupsidedowncakeItem.name=Ananasowe Ciasto Do góry nogami
+item.pineapplehamItem.name=Szynka z Ananasem
+item.pineappleyogurtItem.name=Jogurt Ananasowy
+        
+item.turnipsoupItem.name=Zupa Rzepakowa
+item.roastedrootveggiemedleyItem.name=Mieszanka Pieczonych Bulw
+item.bakedturnipsItem.name=Pieczone Rzepy
+        
+item.gingerbreadItem.name=Piernik
+item.gingersnapsItem.name=Ciasteczka Imbirowe
+item.candiedgingerItem.name=Kandyzowany Imbir
+        
+item.mustardItem.name=Musztarda
+item.softpretzelandmustardItem.name=Precel z Musztardą
+item.spicymustardporkItem.name=Ostra Musztardowa Wieprzowina
+item.spicygreensItem.name=Warzywa na Ostro
+        
+item.garlicbreadItem.name=Chleb Czosnkowy
+item.garlicmashedpotatoesItem.name=Tłuczone Ziemniaki z Czosnkiem
+item.garlicchickenItem.name=Czosnkowy Kurczak
+        
+item.summerradishsaladItem.name=Letnia Sałatka Rzodkiewkowa
+item.summersquashwithradishItem.name=Letni Kabaczek z Rzodkiewką
+        
+item.celeryandpeanutbutterItem.name=Seler z Masłem Orzechowym
+item.chickencelerycasseroleItem.name=Zapiekanka z Kurczakiem i Selerem
+item.peasandceleryItem.name=Groszek i Seler
+item.celerysoupItem.name=Zupa Selerowa
+        
+item.zucchinibreadItem.name=Chleb Cukiniowy
+item.zucchinifriesItem.name=Frytki z Cukinii
+item.zestyzucchiniItem.name=Cukinia na Ostro
+item.zucchinibakeItem.name=Zapiekanka z Cukinii
+        
+item.asparagusquicheItem.name=Szparagowe Quiche
+item.asparagussoupItem.name=Zupa Szparagowa
+        
+item.walnutraisinbreadItem.name=Chleb z Orzechami i Rodzynkami
+item.candiedwalnutsItem.name=Kandyzowane Orzechy
+item.brownieItem.name=Brownie
+        
+item.papayajuiceItem.name=Sok z Papai
+item.papayasmoothieItem.name=Smoothie z Papai
+item.papayayogurtItem.name=Yogurt z Papai
+        
+item.starfruitjuiceItem.name=Sok z Gwiezdnego Owocu
+item.starfruitsmoothieItem.name=Smoothie z Gwiezdnego Owocu
+item.starfruityogurtItem.name=Yogurt z Gwiezdnego Owocu
+        
+item.guacamoleItem.name=Guacamole
+item.creamofavocadosoupItem.name=Kremowa Zupa z Awokado
+item.avocadoburritoItem.name=Burrito z Awokado
+        
+item.poachedpearItem.name=Gotowana Gruszka
+item.fruitcrumbleItem.name=Kruszonka z Owocami
+item.pearyogurtItem.name=Gruszkowy Jogurt
+item.plumyogurtItem.name=Śliwkowy Jogurt
+        
+item.bananasplitItem.name=Bananowy Split
+item.bananajuiceItem.name=Sok Bananowy
+item.banananutbreadItem.name=Chleb Bananowo-Orzechowy
+item.bananasmoothieItem.name=Bananowe Smoothie
+item.bananayogurtItem.name=Bananowy Jogurt
```

Comment:
a tu już dobrze jogurt :p
słowo jogurt powinno zaczynać nazwę (najpierw danie potem smak, są oczywiście wyjątki), więc w takim przypadku będzie trzeba poprzestawiać parę słów w różnych tłumaczeniach  ale nie oznaczam ich na review

Reply:
Najpierw danie potem smak jest jakąś konwencją? Dla mnie lepiej w większości przypadków brzmiało smak-danie. Pomijając to, że miejscami jest tak, a miejscami tak, ujednolicę to 😄

Reply:
generalnie tak się mówi, ale są pojedyncze wyjątki więc jak coś bardzo nie pasuje to pewnie jest git, tak samo stan przed daniem czyli np. pieczone ciasto czekoladowe (tylko jako przykład ciasto jest w domyśle pieczone)